packages/table/doc/basic.vue

```diff
@@ -0,0 +1,46 @@
+<template>
```

Comment:
We can simply remove this folder `packages/table/doc/*`, we are now using the doc website to dev nad proview.